app/components/HomePage/Careers.tsx

```diff
@@ -68,14 +72,17 @@ export default function Careers(props: PageHomeBlocksCareerSection) {
           />
         </Box>
         <Box mx="auto">
-          <Button
-            variant="mc-red"
-            w="250px"
-            position={{ base: "static", lg: "relative" }}
-            top={{ base: "0", lg: "85" }}
-          >
-            {props.buttonLabel}
-          </Button>
+          <Link href="/careers" passHref>
+            <Button
+              as="a"
```

Comment:
Since you added the Link wrapper around the button, there is an <a> tag within an <a> tag and it is causing the following error:

"on-recoverable-error.js:21 Uncaught Error: Hydration failed because the initial UI does not match what was rendered on the server.

Warning: Expected server HTML to contain a matching <a> in <a>."

Removing line 77 will fix the problem.

Reply:
Done ✅